assets/js/blocks/cart-checkout/checkout/checkout-order-error/index.js

```diff
@@ -48,7 +49,11 @@ const CheckoutError = () => {
 const ErrorTitle = ( { errorData } ) => {
 	let heading = __( 'Checkout error', 'woo-gutenberg-products-block' );
 
-	if ( errorData.code === 'woocommerce_product_out_of_stock' ) {
+	if (
+		errorData.code === 'woocommerce_product_out_of_stock' ||
+		errorData.code === 'woocommerce_rest_cart_product_is_not_purchasable' ||
+		errorData.code === 'woocommerce_rest_cart_product_no_stock'
```

Comment:
I wonder if we should define (and export) constants for these error types?

Reply:
What would the benefits be of doing that?  Is this like the Action Names in wp.data being constants? Also not sure of the benefits of that either since the constant names are identical to the contents.

Reply:
the point of constants is that you can’t typo them, and you can autocomplete them, if you do a typo, it will throw an error.
for this however, I think the whole expression can be moved to a variable since it’s the same three times

Reply:
> What would the benefits be of doing that?

As Nadir mentioned, biggest benefit is code auto-completion and preventing typos. This will be useful when we add test coverage for this as well because we can use the constants (again for auto-complete and to remove typos).

> Is this like the Action Names in wp.data being constants? Also not sure of the benefits of that either since the constant names are identical to the contents.

In the data stores the constants are referenced in the action creators and again in the reducers. So having the constants helps with code auto-completion and to guard against typos.